metadata/src/main/java/org/apache/kafka/controller/QuorumControllerMetrics.java

```diff
@@ -276,12 +297,15 @@ public long lastAppliedRecordTimestamp() {
     public void close() {
         Arrays.asList(
             ACTIVE_CONTROLLER_COUNT,
+            FENCED_BROKER_COUNT,
```

Comment:
Fixing the corresponding test surfaced the fact that we were not removing these metrics from the registry. Not sure if that was on purpose.